extensions/amp-inline-gallery/0.1/amp-inline-gallery-captions.css

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-inline-gallery-captions {
+  pointer-events: none;
```

Comment:
It's because this implementation doesn't actually have any descendants in the `amp-inline-gallery-captions` element, which is effectively just a fixed height block of space that gets covered by captions elements in a different tree (`amp-inline-gallery-slide`). Agreed it is a bit unconventional and likely not necessary, but it should not be harmful.

```html
<amp-inline-gallery layout="container">
    <amp-base-carousel>
      <amp-inline-gallery-slide>
        <amp-img></amp-img>
        <!-- clickable and slotted into an amp-truncate-text element added to amp-inline-gallery-slide -->
        <span class="caption" slot="caption"> 
          The first slide's caption.
        </span>
        <span class="attribution" slot="attribution">This is an attribution (not truncated).</span>
      </amp-inline-gallery-slide>
      ... 
    </amp-base-carousel>
    <amp-inline-gallery-captions layout="fixed-height" height="3.75em">
    </amp-inline-gallery-captions>
  </amp-inline-gallery>
```